packages/dd-trace/test/appsec/iast/overhead-controller.spec.js

```diff
@@ -177,13 +178,23 @@ describe('Overhead controller', () => {
   })
   describe('full feature', () => {
     describe('multiple request at same time', () => {
-      function app () {
+      const requestStartedChannel = dc.channel('overhead-controller:test:request:started')
```

Comment:
Why use diagnostics channel for this? It doesn't quite seem like the right tool for the job.